### Get backend chess engine's move service.

---
ChessMove move   # engine's fully quantified move
